### DISCUSSION TITLE
Option to sort by recently usage

### DISCUSSION THREAD

**Discussion by chiawendt** [Authenticator-Extension/Authenticator#874] [Ideas]
When this option is enabled, the entries will be sorted by time of last usage.

**Comment by adelara** [comment]
That could probably solve an annoyance that makes the authenticator randomly closing and only exhibiting one entry.... but I still rather have the option to sort and leave it as I sorted: it's counterintuitive to keep doing auto-sort as your brain will remember where it was, not where it will sorted to.
So... if this comes as feature, please allow me to disable it. Thanks.
